portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,7 +39,14 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    int maxComments = Integer.parseInt(request.getParameter("maxComments"));
+    int maxComments;
+    String maxCommentsParam = request.getParameter("maxComments");
+    if (maxCommentsParam == null){
```

Comment:
Two nits (fix in some other PR): space after closing paranthese + else should be at the same line of the closing bracket for if:
```
if (...) {
  ...
} else {
  ...
}
```

Also, not about why you do this even though your client now always sends maxComments: 
The server should never rely on the client's behavior. It should always protect itself from throwing unexpected exceptions that result from invalid client behavior. So even if your client is now behaving nicely, this doesn't mean that there won't be a future case in which the client will send a faulty request.

Reply:
I understand :)